modules/custom/az_migration/config/install/migrate_plus.migration.az_person_categories.yml

```diff
@@ -0,0 +1,41 @@
+id: az_person_categories
+label: Person Categories
+migration_group: az_migration
+status: true
+
+source:
+  plugin: az_d7_taxonomy_term
```

Comment:
The standard `d7_taxonomy_term` plugin seems like it could be used here.

Reply:
yes, I agree. I will update it. Thanks